lib/server.js

```diff
@@ -61,6 +61,7 @@ GeneralRouter.get('/', events.listEvents);
 GeneralRouter.use(middlewares.ensureAuthorized);
 GeneralRouter.post('/', events.addEvent);
 GeneralRouter.get('/tasks', middlewares.getTasksList);
+GeneralRouter.get('/mine/participating', events.listUserAppliedEvents);
```

Comment:
I suggest renaming it to `/events/mine`, oms-events has slightly different endpoints structure, there's no events list you are organizing